addons/l10n_it/models/account_tax.py

```diff
@@ -2,6 +2,7 @@
 
 from odoo import _, api, fields, models
 from odoo.exceptions import ValidationError, UserError
+from odoo.tools.translate import get_text_content
```

Comment:
```suggestion
from odoo.tools import html2plaintext
```
I think `html2plaintext` is more commonly used for this.